CPPLINT.cfg

```diff
@@ -1,13 +1,13 @@
 set noparent
 headers=h,hpp,hxx
+extensions=cpp,h,hpp,hxx
 root=sources
 linelength=160
 
 # Filter checks
-filter=-legal/copyright
+filter=-build/c++11
 filter=-build/include
 filter=-build/include_what_you_use
 filter=-build/include_order
-filter=-build/c++11
 filter=-runtime/references
-filter=-runtime/string
+filter=-runtime/int
```

Comment:
Why did you add this one ? verboten C basic types should not be used

Reply:
It comes from the configuration used in dynawo. I just copied the current CPPLINT.cfg found in master branch of dynawo. Should this filter be removed here and also in dynawo ?

Reply:
In dynawo, it shouldn't. But in DFL, maybe it should because we have less constraints. @gautierbureau what do you think ?

Reply:
After looking at the definitions of the filters, runtime/string and runtime/int should be present in cpplint of DFL

Reply:
Done, the line for runtime/string was recovered

Reply:
No it is the opposide, the filters should be checked. Your lines removed the check

Reply:
Sorry for the confusion. I updated CPPLINT.cfg so both filters are not discarded.